csharp-basic-topics/MemoryStreamInCsharp/Tests/Tests/Tests.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.IO;
+using Xunit;
+
+namespace Tests;
+
+public class Tests
+{
+    [Fact]
+    public void WhenCreateMemoryStream_ThanCanWrite()
+    {
+        var memoryStream = new MemoryStream();
+        Assert.True(memoryStream.CanWrite);
+    }
+
+    [Fact]
+    public void WhenWritingToExtendibleMemoryStream_ThenCanWrite()
```

Comment:
In the name of the test method, in the end, it says `ThenCanWrite, but that's not actually what we are testing/asserting here. It should say something like `ThenLengthIsCorrect`